django/urls/converters.py

```diff
@@ -53,6 +56,15 @@ class PathConverter(StringConverter):
 
 
 def register_converter(converter, type_name):
+    if type_name in REGISTERED_CONVERTERS or type_name in DEFAULT_CONVERTERS:
```

Comment:
Just fyi (optional "improvement" - depending on taste I guess) we can do set union:
```suggestion
    if type_name in REGISTERED_CONVERTERS | DEFAULT_CONVERTERS:
```

If find it slightly easier to read as your mind's "python interpreter" doesn't need to form 2 expressions?  Also makes line a little shorter 😊

Reply:
This can be way slower as it does an O(n) merge before checking for membership, rather than two O(1) checks. In practice it won't matter unless a project has hundreds of path converters, but I'd rather not use a slow pattern where a faster one is readily available.

Reply:
Yup 👍